bundles/org.openhab.binding.bluetooth.daikinmadoka/pom.xml

```
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+	<modelVersion>4.0.0</modelVersion>
+
+	<parent>
+		<groupId>org.openhab.addons.bundles</groupId>
+		<artifactId>org.openhab.addons.reactor.bundles</artifactId>
+		<version>2.5.6-SNAPSHOT</version>
```

Sorry that we missed 2.5.6 - please hence update to 2.5.7, which should make the build green again.
```suggestion
		<version>2.5.7-SNAPSHOT</version>
```

---

Upgraded to 2.5.7. Compiles fine including tests.